src/damage.cpp

```diff
@@ -243,7 +247,8 @@ int aoe_size( const std::set<std::string> &tags )
                tags.count( "FRAG" ) ) {
         return 2;
     } else if( tags.count( "ACIDBOMB" ) ||
-               tags.count( "FLAME" ) ) {
+               tags.count( "FLAME" ) ||
+               tags.count( "GAS_FUNGICIDAL" ) ) {
```

Comment:
`aoe_size` is currently only used for vehicles to avoid damaging themselves. It doesn't have other tags like `"TEARGAS"`, so it probably shouldn't have `"GAS_FUNGICIDAL"` either, at least for now.

For toxic gases, some more specialized function will later exist (one that checks user's resistance).